### PR TITLE
[introduction-to-codeql] - resolves template not loading when starting course

### PR DESCRIPTION
**Pull request by felickz**
#### Before:
[start-course](https://github.com/new?template_owner=skills&template_name=enable-code-scanning&owner=%40me&name=skills-enable-code-scanning&description=My+clone+repository&visibility=public)

image

#### After:
[start-course](https://github.com/new?template_owner=skills&template_name=introduction-to-codeql&owner=%40me&name=skills-enable-code-scanning&description=GitHub+Skills:+Enable+CodeQL&visibility=public)

image

### Changes

Fixes link in start course

Closes:

### Task list

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Code scanning is part of the [GitHub Advanced Security (GHAS)](https://docs.gith
 }).toString()
 -->
 
-[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=enable-code-scanning&owner=%40me&name=skills-enable-code-scanning&description=My+clone+repository&visibility=public)
+[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=introduction-to-codeql&owner=%40me&name=introduction-to-codeql&description=My+clone+repository&visibility=public)
```
[the remark this answers is not in the document]
Comment (leftrightleft) [reply README.md:54]: My change edits the description to reflect the name of the Skill

[the remark this answers is not in the document]
Comment (felickz) [reply README.md]: fixed!

File: README.md [hunk 2]
```diff
@@ -51,8 +51,7 @@ Code scanning is part of the [GitHub Advanced Security (GHAS)](https://docs.gith
 }).toString()
 -->
 
-[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=enable-code-scanning&owner=%40me&name=skills-enable-code-scanning&description=My+clone+repository&visibility=public)
-
+[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=introduction-to-codeql&owner=%40me&name=introduction-to-codeql&description=GitHub+Skills:+Enable+CodeQL&visibility=public)
```
Comment (hectorsector) [inline README.md:54]:
```suggestion
[![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=introduction-to-codeql&owner=%40me&name=skills-introduction-to-codeql&description=GitHub+Skills:+Enable+CodeQL&visibility=public)
```

**Review by hectorsector — APPROVED** [review]
The suggestion prefixes the user's repo with `skills-` which makes them easier to spot in their repo list. Other than that, looks good to me!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (felickz) [reply README.md]: Fixed

**Comment by hectorsector** [comment]
Quick heads up on 8ac9691, after this commit the name of the user's repo will actually not match the name of the skill. I don't think that's intended so wanted to flag it for y'all.

**Comment by leftrightleft** [comment]
> Quick heads up on [8ac9691](https://github.com/skills/introduction-to-codeql/commit/8ac9691600a956dd5c204f9dd8ccca27c377f930), after this commit the name of the user's repo will actually not match the name of the skill. I don't think that's intended so wanted to flag it for y'all.

Just fixed.  Thanks @hectorsector

**Pull request merged**